ee/api/test/test_feature_flag_role_access.py

```diff
@@ -0,0 +1,107 @@
+from rest_framework import status
+
+from ee.api.test.base import APILicensedTest
+from ee.models.feature_flag_role_access import FeatureFlagRoleAccess
+from ee.models.organization_resource_access import OrganizationResourceAccess
+from ee.models.role import Role
+from posthog.models.feature_flag import FeatureFlag
+from posthog.models.organization import OrganizationMembership
+from posthog.models.user import User
+
+
+class TestFeatureFlagRoleAccessAPI(APILicensedTest):
+    def setUp(self):
+        super().setUp()
+        self.eng_role = Role.objects.create(name="Engineering", organization=self.organization)
+        self.marketing_role = Role.objects.create(name="Marketing", organization=self.organization)
+        self.feature_flag = FeatureFlag.objects.create(
+            created_by=self.user, team=self.team, key="flag_role_access", name="Flag role access"
+        )
+
+    def test_can_always_add_role_access_if_creator_of_feature_flag(self):
+        OrganizationResourceAccess.objects.create(
+            resource=OrganizationResourceAccess.Resources.FEATURE_FLAGS,
+            access_level=OrganizationResourceAccess.AccessLevel.DEFAULT_VIEW_ALLOW_EDIT_BASED_ON_ROLE,
```

Comment:
shouldn't this be `ONLY_VIEW` for this test to make sense?

Reply:
i.e. this person can ONLY_VIEW flags, except for the flags they created, where they can add other roles.